codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -457,14 +457,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-deploy-plugin</artifactId>
-        <version>2.7</version>
+        <version>2.8.2</version>
         <configuration>
           <retryFailedDeploymentCount>10</retryFailedDeploymentCount>
         </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-release-plugin</artifactId>
-        <version>2.4.1</version>
+        <version>2.5.3</version>
```

Comment:
I tried to upgrade maven-release-plugin to a newer version than 2.4.1 some time ago, but it blocked me from releasing due to bad interaction with git. I'm unsure the devs fixed the problem. (We will see when we release another version?)